File: src/main/java/destiny/tools/screen/Child.java
package destiny.tools.screen;
/**
 * @author smallufo
 * @date 2002/8/29
 * @time 上午 02:23:42
 */
class Child
{
  private Canvas c;
  private int x;
  private int y;

  public Child(Canvas c , int x , int y)
  {
    this.c = c;
    this.x = x;
    this.y = y;
  }
  
  public void setParent(Canvas c)
  {
    this.c.setParent(c);
  }
  
  public Canvas getCanvas()
  {
    return this.c;
  }
  
  public int getX() { return this.x; }
  public int getY() { return this.y; }
}

File: src/main/java/destiny/tools/screen/Child.kt
package destiny.tools.screen

/**
 * @author smallufo
 * @date 2002/8/29
 * @time 上午 02:23:42
 */
internal class Child(val canvas: Canvas, val x: Int, val y: Int) {

  fun setParent(c: Canvas) {
    this.canvas.parent = c
  }
}
